derive_builder_test/tests/run-pass/try_setter.rs

```diff
@@ -1,11 +1,11 @@
-#![cfg_attr(feature = "try_setter", feature(try_from))]
+#![cfg_attr(feature = "nightlytests", feature(try_from))]
 
 #[macro_use]
 extern crate derive_builder;
 
 #[allow(unused_imports)]
 mod struct_level {
-    #[cfg(feature = "try_setter")]
+    #[cfg(feature = "try_from")]
```

Comment:
ok, gating on `try_from` instead of `nightlytests` is actually a good idea here. Please do it consistently in the rest of the test then - that'd be easier to understand. :-)

Alternatively also gate on `nightlytests` here.